data.js

```diff
@@ -63,7 +63,17 @@ const emojiItems = [
         musicVideo: "https://www.youtube.com/watch?v=vc6vs-l5dkc",
         genres: ["emo", "pop", "punk"],
         year: 2005
-    },{
+    },
+    {
+        title: "I'll Make a Man Out of You",
+        artist: [" Donny Osmond"],
+        featuredArtist: [],
+        emojiImgs: "9️⃣🕒🌆",
```

Comment:
Please update the emojis for this song, it looks like you might have used a different song as a template and haven't updated the emojis yet.

Reply:
Okay I will update the changes required and make a PR for the same